notebooks/20220913-fraction-mtx-after-subtracting-mgx.ipynb

```diff
@@ -0,0 +1,555 @@
+{
```

Comment:
<ul><li>what is the impact of downsampling to 1/200 here? If I understand correctly, you are comparing 0.05% of the metagenome to 0.05% of the transcriptome. assuming the full transcriptome and the full metagenome are actually exactly the same, what would the baseline expectation be that you would see something in the downsampled transcriptome not represented in the downsampled metagenome?  </li><li>for downstream assembly will the reads need to be adapter trimmed? </li><li>will you do the assembly on the downsampled and subtracted k-mers ? </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/2022-mtx-not-in-mgx-pairs/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
1. it would be comparing 0.5%, but yes. I added more details to the notebook describing what the downsampling value is doing here. Because the same sequences are retained across samples, if a full transcriptome and full metagenome were exactly the same, there would be no sequences in the downsampled transcriptome that weren't in the metagenome -- the sketches would be exactly the same.
2. Yes for downstream analysis I will adapter and k-mer trim the reads ( I don't plan to do assembly, but will build assembly graphs; assembly graphs will have better recall for things that are systematically not in our databases, which is what we're targeting here). It will be fast to intersect the fractions of the the leftover sketches that are retained in the trimmed reads.
3. No I won't do assembly with the downsampled and subtracted k-mers. Because they're downsampled, it becomes effectively meaningless to do that type of analysis with them (i've done it multiple times for multiple projects over the last 5 years and it's always very disappointing). Instead, I'm using this analysis to inform what downstream samples to dig into more with assembly graph methods.